### ISSUE TITLE
Overflow auto breaks scrolling focus on cursor

### ISSUE THREAD

**Issue by stefanneculai** [froala/wysiwyg-editor#2981]
Details on https://github.com/froala/angular-froala-wysiwyg/issues/221#issuecomment-406638913.

**Comment by Huuuze** [comment]
I'd like to know the ETA for a fix on this issue. It's really annoying.

**Comment by stefanneculai** [comment]
We don't think it's right to set expectations that may not be met due to shifting schedules. Therefore, we can't release any information about when this may be fixed for now.

**Comment by Huuuze** [comment]
@stefanneculai, fair enough. Please know that this issue is having a directly operational impact for our customers. I hope the issue is prioritized accordingly.

**Comment by stefanneculai** [comment]
We just reviewed this issue and it is more extensive than it looks initially. It would probably be addressed in release 2.9.0 or 2.10.0.

**Comment by Huuuze** [comment]
@stefanneculai - do 2.9.0 or 2.10.0 have scheduled release dates?

**Comment by Huuuze** [comment]
@stefanneculai - I'm a little concerned since Froala has not received an update in nearly 3 months.

**Comment by Mboulianne** [comment]
It's alright for me since we found a workaround (overflow: hidden scrollTop: 0 the body when opening the fixed container)

**Comment by ilyaskarim** [comment]
We’re closing this issue due to inactivity. If you’re still experiencing this problem on the latest version of our editor, please feel free to reopen the issue and let us know. You can find details on the latest updates here: [Froala Editor Changelog](https://froala.com/wysiwyg-editor/changelog/).
